### PR TITLE
BLADE-348 distributionUrl 301 Moved Permanently

### PR DESCRIPTION
**Pull request by vsingleton**

### CODE REVIEW

File: cli/test-resources/projects/testws1/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.5.1-bin.zip
```
Comment (gamerson) [inline cli/test-resources/projects/testws1/gradle/wrapper/gradle-wrapper.properties]: Thank you vernon!

**Comment by vsingleton** [comment]
Hope it helps.  The test looked like it failed bc it was still hitting the services sub domain.  Weird.

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-blade-cli/pull/604. [See changes here.](https://github.com/gamerson/liferay-blade-cli/compare/986c2249~1...986c2249) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/christopherbryanboyd/liferay-blade-cli/pull/17. [See changes here.](https://github.com/gamerson/liferay-blade-cli/compare/4ba11cba~1...4ba11cba) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**